app.js

```diff
@@ -394,6 +446,11 @@ function search() {
 }
 
 function onInput(e) {
+  var key = e.target.name || e.target.id;
```

Comment:
```html
<input id="good-first" value="good-first" checked="" type="checkbox">
<input id="mentored" value="mentored" type="checkbox">
```

these inputs isn't named, so I just use `ids`, is it ok to add a name attribute?